index.md

```diff
@@ -8,6 +8,573 @@ This is some welcome text.
+same.
+
+```python
+class Parent:
+	def __init__(self):
+		self.num = 2
+	
+	def return_num(self, multiple):
+		return self.num * multiple
+
+class Child(Parent):
+	def __init__(self)
+		Parent.__init__(self)
+	
+	def return_num(self):  # Missing argument (to keep signature identical)
+		return 42
+```
+
+
+
+### W0231: Super init not called {#W0231}
+
+When inheriting from a parent, you need to call the parent's _init_
+method using itself as a parameter (or use _super_ if you are more
+familiar with that). The whole goal of extending a class is to be
+a child of the class you extend from, and properties that the parent
+sets in its constructor would not be propagated into the child you are
+creating.
+
+Therefore you must always call the parent.
+
+```python
+class Parent:
+	def __init__(self):
+		self.num = 1
+
+class Child(Parent):
+	def __init__(self):
+		Parent.__init__(self)  # You must have this
+```
+
+
+
+### W0233: Bad parent init {#W0233}
+
+You should call the _init_ method of the parent, not some arbitrary and
+unrelated class. To fix this, use the _init_ from the parent of the class
+you are inheriting from.
+
+```python
+class ClassA:
+	def __init__(self):
+		pass
+
+Class Parent:
+	def __init__(self):
+		pass
+
+class Child(Parent):
+	def __init__(self):
+		ClassA.__init__(self)  # Not a child of class A
+```
+
+
+
+### W0702: Bare exception {#W0702}
+
+While you can catch any exception without specifying a class to catch after
+the _except_ keyword, this is bad practice since it will mask other exceptions
+that could appear which you do not want to catch, making your code brittle.
+Always provide the classes you expect to catch.
+
+```python
+def no_catching():
+	try:
+		raise TypeError()
+	except:
+		print('Requires an exception class')
+```
+
+
+
+### W0703: Exception is too generic {#W0703}
+
+Similar to exception error W0702, if your exception is too generic then you
+may end up burying errors since they will be always caught. Because of this,
+you should not be using _except Exception_ as your except block since it can
+catch exceptions you do not want caught.
+
+```python
+def generic_catch():
+	try:
+		a = 5 / 0
+	except Exception:
+		print('Got exception')
+```
+
+
+
+### W0705: Duplicate except blocks {#W0705}
+
+When you list an exception to be caught, you should not list it again. Only
+have one block for each exception. This does not include having child and 
+parent exceptions, but rather it means don't have _exception MyException_,
+and then later on when listing exceptions you have _exception MyException_
+again.
+
+```python
+def repeat_except_blocks():
+	try:
+		raise Exception()
+	except Exception:
+		print('This is triggered')
+	except Exception:
+		print('Duplicate exception block')
+```
+
+
+
+### W0711: Binary op exception {#W0711}
+
+The proper way to have an except block catch multiple exceptions is to have
+the exception classes in a tuple. You can write an except block in Python
```

Comment:
Say "Python allows you to ..., but ..." to make it clear from the beginning of the sentence that you shouldn't do it.